frontend/package.json

```diff
@@ -4,7 +4,7 @@
     "private": true,
     "dependencies": {
         "@babel/runtime": "7.4.2",
-        "@cthit/react-digit-components": "^1.0.0-beta.41",
+        "@cthit/react-digit-components": "^1.0.0-beta.42",
```

Comment:
Why not the latest version (52)?

Reply:
Updated to latest version in latest commit